flixel/system/frontEnds/HTML5FrontEnd.hx

```diff
@@ -59,6 +61,49 @@ class HTML5FrontEnd
 	{
 		return Browser.window.innerHeight;
 	}
+	
+	private inline function get_platform():FlxPlatform
```

Comment:
We could probably just run this entire thing once in `new()` and cache the result? It's not like it's gonna change at runtime, right? :)